### PR TITLE
Video bump

### PR DESCRIPTION
**Pull request by lukesneeringer**
This bumps video intelligence to 1.0.

Release notes being staged in the releases UI.

### CODE REVIEW

**Comment by dhermes** [comment]
@lukesneeringer Can you also update the following files:

```
CHANGELOG.md
docs/releases.rst
videointelligence/CHANGELOG.md
docs/videointelligence/releases.rst
```

File: videointelligence/setup.py [hunk 1]
```diff
@@ -50,7 +50,7 @@
     zip_safe=False,
     scripts=[],
     classifiers=[
-        'Development Status :: 4 - Beta',
+        'Development Status :: 5 - Production/Stable',
```
Comment (dhermes) [inline videointelligence/setup.py]: Eek

File: videointelligence/CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,31 @@
 
 [1]: https://pypi.org/project/google-cloud-videointelligence/#history
 
+## 1.0.0
+
+[![release level](https://img.shields.io/badge/release%20level-general%20availability%20%28GA%29-brightgreen.svg?style&#x3D;flat)](https://cloud.google.com/terms/launch-stages)
```
Comment (dhermes) [inline videointelligence/CHANGELOG.md]: So fancy!

Comment (lukesneeringer) [reply videointelligence/CHANGELOG.md]: Stole it from Node. :-)

**Pull request merged**